CHANGELOG.md

```diff
@@ -1,9 +1,10 @@
 # Changelog
 
-## Unreleased 
+## 20.10.0
 
 ### Features Added
 - Added "eth_signTransaction" JSON RPC
+- Docker namespace updated to reflect "quorumengineering"
```

Comment:
This just got changed to consensys

Reply:
really?

Reply:
yeah it's now consensys/quorum-ethsigner